src/AirshipBindings.iOS.common/build.gradle

```diff
@@ -33,3 +33,42 @@ task build {
 }
 
 build.dependsOn('carthageUpdate')
+
+task carthageCheckout {
+    dependsOn 'syncVersion'
+    doLast() {
+        exec {
+            workingDir "$rootDir"
+            commandLine "carthage", "checkout"
+        }
+    }
+}
+
+task checkoutXamboni {
+    doLast() {
+        // Xamboni
+        def tmpDir = System.getProperty("java.io.tmpdir")
+        def xamboniTempRepo = new File("${tmpDir}/xamboni")
+        project.delete(files(xamboniTempRepo))
+
+        xamboniTempRepo.deleteOnExit()
+
+        println("Cloning Xamboni repo")
+        exec {
+            commandLine "git", "clone", "git@github.com:urbanairship/xamboni.git", "${xamboniTempRepo}"
```

Comment:
Why are we still using git here? Does this task need to be removed?

Reply:
I'm still cloning the Xamboni repo to get Xamboni. Did you have a different idea?

Reply:
Oh, sorry,  I misread that and thought it was cloning the iOS repo.